astrokat/simulate.py

```diff
@@ -365,21 +369,29 @@ def _target_azel(self, target):
             The elevation co-ordinate of the target in degrees.
 
         """
-        az, el = target.azel(simobserver.date)
+        if isinstance(target, dict):
+            scan_target = target.get('ants')
```

Comment:
I think this simulation function is shared between tracks and scans? Referencing 'scan_targets' here is perhaps misleading.